app/src/main/java/mod/tyron/compiler/Util.java

```diff
@@ -0,0 +1,7 @@
+package mod.tyron.compiler;
+
+import java.io.File;
+
+public class Util {
+
+}
```

Comment:
What's this class for?

Reply:
Unused class, it will be used in the future ill remove it on this commit